### PR TITLE
Text utils

### PR DESCRIPTION
**Pull request by lannisterrr**
recording

Added text utility classes

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code seems fine and structured

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

File: index.html [hunk 1]
```diff
@@ -588,6 +588,53 @@ <h3 class="heading-3 t-c-3">Range Slider</h3>
...
+            <h3 class="heading-3 t-c-3">Primary Color Heading</h3>
+            <p class="heading-1">Heading-1</p>
+            <p class="heading-2">Heading-2</p>
+            <p class="heading-3">Heading-3</p>
+          </div>
+          <div class="code-snippet centered-row">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=sharp&l=htmlmixed&width=700&ds=false&dsyoff=20px&dsblur=81px&wc=true&wa=false&pv=36px&ph=51px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522heading-3%2520t-c-3%2522%253EPrimary%2520Color%2520Heading%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522heading-1%2522%253EHeading-1%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522heading-2%2522%253EHeading-2%253C%252Fp%253E%250A%2509%2509%2520%2520%253Cp%2520class%253D%2522heading-3%2522%253EHeading-3%253C%252Fp%253E"
+              style="
+                width: 693px;
```
Comment (Varun-28) [inline index.html:603]: I think it is mentioned in LMS that not to use Inline styles, check on LMS once. Rest is good to go 👉

Comment (lannisterrr) [reply index.html:603]: these styles are from the auto generated code-snippet.

**Pull request merged**